packages/fetch-api/src/api.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable no-dupe-class-members */
-import cookie from 'cookie'
+import { getPingbackId } from '@giphy/js-util'
```

Comment:
@cosmocochrane1 you can get the pingback id in web app like so

Reply:
👍